actors/builtin/miner/miner_test.go

```diff
@@ -1830,8 +1927,21 @@ func (h *actorHarness) onProvingPeriodCron(rt *mock.Runtime, expectedEnrollment
 		},
 		exitcode.Ok)
 
-	{
-		// Detect and penalise missing faults (not yet implemented)
+	if len(undetectedFaultySectors) > 0 {
+		// Process ongoing faulty sectors (not yet implemented)
```

Comment:
This phrasing is confusing, perhaps add a TODO maybe with a github issue attached.